.github/workflows/ci-build.yml

```diff
@@ -55,9 +55,6 @@ jobs:
     - name: Install local lsp
       run: npm i ../fabric8-analytics-lsp-server-*.tgz
       working-directory: ./vscode-extension
-    - name: Copy fabric8-analytics-lsp-server files to one level up(FIXME)
```

Comment:
This was added a workaround during the release package refactoring. It is no longer needed.